packages/contracts/contracts/voting/majority/IMajorityVoting.sol

```diff
@@ -88,56 +144,50 @@ interface IMajorityVoting {
     /// @param execResults The bytes array resulting from the proposal execution in the associated DAO.
     event ProposalExecuted(uint256 indexed proposalId, bytes[] execResults);
 
-    /// @notice Emitted when the vote settings are updated.
-    /// @param supportThreshold The support threshold in percent.
-    /// @param minParticipation The minimum participation ratio in percent.
-    /// @param minDuration The minimal duration of a vote in seconds.
-    /// @param minProposerVotingPower The minimal voting power needed to create a proposal.
-    event VoteSettingsUpdated(
+    /// @notice Emitted when the plugin settings are updated.
+    /// @param voteMode A parameter to select the vote mode.
+    /// @param supportThreshold The support threshold value.
+    /// @param minParticipation The minimum participation value.
+    /// @param minDuration The minimum duration of the proposal vote in seconds.
+    /// @param minProposerVotingPower The minimum voting power required to create a proposal.
+    event PluginSettingsUpdated(
+        VoteMode voteMode,
```

Comment:
Voting mode sounds like a parameter that you can choose upon proposal creation, not upon plugin installation.

Maybe it's ok like this, just pointing out

Reply:
I named it to `VotingMode` and I would like to keep it unless we find something better.